ckanext/nextgeoss/templates/package/search.html

```diff
@@ -162,7 +162,8 @@ <h2 style="display: flex; justify-content: center;">Error</h2>
         {% endif %}
       {% endblock %}
       {% block package_search_results_list %}
-        {{ h.snippet('snippets/package_list.html', packages=c.page.items) }}
+        {% set noa_packages_resources = h.get_noa_linker_resources(c.page.items) %}
```

Comment:
This needs to happen here so that all the packages are processed in parallel at the same time.